src/components/NewGalaxyModal.js

```diff
@@ -0,0 +1,91 @@
+import _ from 'lodash/fp'
+import PropTypes from 'prop-types'
+import { Component, Fragment } from 'react'
+import { div, h, li, p, span, ul } from 'react-hyperscript-helpers'
+import { ButtonPrimary, ButtonSecondary, Link } from 'src/components/common'
+import { withModalDrawer } from 'src/components/ModalDrawer'
+import TitleBar from 'src/components/TitleBar'
+import { machineTypes } from 'src/data/machines'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+const styles = {
+  whiteBoxContainer: { padding: '1rem', borderRadius: 3, backgroundColor: 'white' },
+  drawerContent: { display: 'flex', flexDirection: 'column', flex: 1, padding: '1.5rem' }
+}
+
+export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewClusterModal extends Component {
+  static propTypes = {
+    onDismiss: PropTypes.func.isRequired,
+    onSuccess: PropTypes.func.isRequired
+  }
+
+  render() {
+    const { onDismiss, onSuccess } = this.props
+    const createGalaxy = () => {
+      onSuccess()
+      //TODO
+    }
+
+    const bottomButtons = () => {
+      return h(Fragment, [
+        div({ style: { display: 'flex', margin: '3rem 0 1rem' } }, [
+          div({ style: { flex: 1 } }),
+          h(ButtonPrimary, {
+            disabled: true,
+            onClick: () => createGalaxy()
+          }, ['Create']),
+          h(ButtonSecondary, { style: { margin: '0 2rem 0 1rem' }, onClick: onDismiss }, 'Cancel')
+        ])
+      ])
+    }
+
+    const getMachineDetails = () => {
+      return _.filter(({ name }) => name === 'n1-standard-8', machineTypes)[0]
+    }
+
+    return h(Fragment, [
```

Comment:
I might go ahead and create the viewmode thing we did for NewClusterModal, just to follow the same patterns. (We can also do that when we add the second view mode for delete/confirmations)

Reply:
I agree, I will make the update